google/cloud/aiplatform/datasets/column_names_dataset.py

```diff
@@ -237,6 +237,12 @@ def _retrieve_bq_source_columns(
         if bq_table_uri.startswith(prefix):
             bq_table_uri = bq_table_uri[len(prefix) :]
 
+        # The colon-based "project:dataset.table" format is no longer supported:
+        # Invalid dataset ID "bigquery-public-data:chicago_taxi_trips".
+        # Dataset IDs must be alphanumeric (plus underscores and dashes) and must be at most 1024 characters long.
+        # Using dot-based "project.dataset.table" format instead.
+        bq_table_uri.replace(":", ".")
```

Comment:
`.replace()` returns the modified copy of the string, rather than applying changes in place, right? This line has no actual effect on the value of `bq_table_uri`, if I read things correctly.

This should be the correct form?
```suggestion
        bq_table_uri = bq_table_uri.replace(":", ".")
```

Reply:
Thank you!
I'm don't know how I overlooked this bug.